react/react2/week1/github-search/src/components/SearchContext.js

```diff
@@ -0,0 +1,4 @@
+import { createContext } from "react";
+
+export const SearchContext = createContext();
+export const SearchContextProvider = SearchContext.Provider;
```

Comment:
You have just created the context, neither children are passed to context nor values are passed to children!

Reply:
the context values object is defined in the App.js and passed as a value using contextProvider

Reply:
Your code flow might work for now as this is a very simple assignment. The advantage of using React is its components, where we can build the small components separately and connect them through the App.js file.
Going ahead, no fetching code or any logical code is included in the app.js file, it is used to connect all the components in the order we want.